src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.hpp

```diff
@@ -1024,16 +1024,17 @@ class ShenandoahRegionChunkIterator : public StackObj {
   static const size_t _maximum_chunk_size_words = (4 * 1024 * 1024) / HeapWordSize;
 
   static const size_t _clusters_in_smallest_chunk = 4;
-  static const size_t _assumed_words_in_card = 64;
 
   // smallest_chunk_size is 4 clusters (i.e. 128 KiB).  Note that there are 64 words per card and there are 64 cards per
   // cluster.  Each cluster spans 128 KiB.
   // This is computed from CardTable::card_size_in_words() *
   //      ShenandoahCardCluster<ShenandoahDirectCardMarkRememberedSet>::CardsPerCluster;
   // We can't perform this computation here, because of encapsulation and initialization constraints.  We paste
   // the magic number here, and assert that this number matches the intended computation in constructor.
```

Comment:
Is the portion of the comment beginning "We paste the maginc number ... etc." not obsolete now, and should there be deleted?

Reply:
Yes - I'll clean that up.